examples/vault-plugin/demo.registerJSS.sh

```diff
@@ -18,7 +18,7 @@ HELPMEHELPME
 }
 kk="kubectl -n trusted-identity"
 
-# this function registers individualx
+# this function registers individual node
```

Comment:
nit : individual nodes

Reply:
Actually, it's a single node, one by one